docs/installation.md

```diff
@@ -82,7 +82,7 @@ IBM offer academic licenses for CPLEX. Refer to the IBM website for details.
 
 ## Python module requirements
 
-Refer to [requirements.txt](https://github.com/calliope-project/calliope/blob/main/requirements.txt) in the Calliope repository for a full and up-to-date listing of required third-party packages.
+Refer to the [requirements directory](https://github.com/calliope-project/calliope/blob/main/requirements) in the Calliope repository for a full and up-to-date listing of required third-party packages.
```

Comment:
Best to still link to https://github.com/calliope-project/calliope/blob/main/requirements/base.txt since that is what matters when you install Calliope?